data/data-pipeline/tile/generate.py

```diff
@@ -1,63 +1,54 @@
-import os
 from pathlib import Path
-import shutil
+import os
 
-from etl.sources.census.etl_utils import get_state_fips_codes
+from utils import remove_all_from_dir
+from utils import get_module_logger
+
+logger = get_module_logger(__name__)
 
 
 def generate_tiles(data_path: Path) -> None:
 
+    score_tiles_path = data_path / "score" / "tiles"
+    high_tile_path = score_tiles_path / "high"
+    low_tile_path = score_tiles_path / "low"
+    score_geojson_dir = data_path / "score" / "geojson"
+
     # remove existing mbtiles file
-    mb_tiles_path = data_path / "tiles" / "block2010.mbtiles"
-    if os.path.exists(mb_tiles_path):
-        os.remove(mb_tiles_path)
+    remove_all_from_dir(score_tiles_path)
+
+    # create dirs
+    os.mkdir(high_tile_path)
+    os.mkdir(low_tile_path)
+
+    # generate high mbtiles file
+    logger.info(f"Generating USA High mbtiles file")
+    cmd = "tippecanoe "
+    cmd += "--force -Z 5 -z 11 -l usa_high "
+    cmd += "-o " + str(high_tile_path) + "/usa_high.mbtiles "
+    cmd += str(score_geojson_dir / "usa-high.json")
+    os.system(cmd)
 
-    # remove existing mvt directory
-    mvt_tiles_path = data_path / "tiles" / "mvt"
-    if os.path.exists(mvt_tiles_path):
-        shutil.rmtree(mvt_tiles_path)
+    # generate high mvts
+    logger.info(f"Generating USA High mvt folders and files")
+    cmd = "tippecanoe "
+    cmd += " --drop-densest-as-needed --no-tile-compression  -zg "
```

Comment:
Might be clearer to include the full version of the args rather than the abbreviated - I was initially surprised to find that `-zg` did what it does, and needed the full version to know.

Relatedly, actually, we may not want `-zg`, and/or may want to explicitly specify a low zoom. 
To generate the current high tiles I ran
```sh
docker run --rm -it -v ${PWD}:/data tippecanoe tippecanoe --no-tile-compression -Z 5 -z 11 -e /data/usa_high_prdc -l blocks /data/usa_merged_prdc.geojson
```
for instance, which makes it more explicit what the bounds are. `-zg` feels a bit automagic to me